### PR TITLE
Refactor/switch mysql to postgres

### PR DESCRIPTION
**Pull request by Swiizyy**

### CODE REVIEW

File: prisma/migrations/20240613093958_initial_migrations/migration.sql [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    "id" VARCHAR(19) NOT NULL,
+    "inviter" VARCHAR(19),
+    "announcementChannel" VARCHAR(19),
+    "announcementMessage" VARCHAR(512),
+    "overviewChannel" VARCHAR(19),
+    "overviewMessage" VARCHAR(19),
+    "birthday_role" VARCHAR(19),
+    "birthday_ping_role" VARCHAR(19),
+    "log_channel" VARCHAR(19),
+    "timezone" INTEGER NOT NULL DEFAULT 0,
```
Comment (nikolaischunk) [inline prisma/migrations/20240613093958_initial_migrations/migration.sql:22]: Do we / Should we use timezone offsets or IANA Timezone Names in the DB?

File: src/lib/types/Augments.d.ts [hunk 2]
```diff
@@ -101,6 +101,7 @@ declare module '@skyra/env-utilities' {
 		REDIS_USERNAME: string;
 
 		DATABASE_URL: string;
+		DIRECT_DATABASE_URL: string;
```
Comment (nikolaischunk) [inline src/lib/types/Augments.d.ts:104]: - [ ] Added to doppler?

**Review by nikolaischunk — COMMENTED** [review]
looking forward to the migration 🤩

File: prisma/migrations/20240613093958_initial_migrations/migration.sql (hunk 1 above)
Comment (Swiizyy) [reply prisma/migrations/20240613093958_initial_migrations/migration.sql:22]: I would say directly the IANA Timezone Names would be easier to manage I think

**Review by nikolaischunk — APPROVED** [review]
Let's go!

**Pull request merged**